reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/method/MethodChecker.scala

```diff
@@ -37,7 +37,9 @@ private[analyze] abstract class BaseMethodChecker extends Checker[MemberInfo, Cl
 
 private[analyze] class ClassMethodChecker extends BaseMethodChecker {
   def check(method: MemberInfo, inclazz: ClassInfo): Option[Problem] = {
-    if (method.isDeferred)
+    if (method.nonBridge && method.nonAccessible)
```

Comment:
The reason why this doesn't compute for me is that `ACC_SYNTHETIC` means we can safely ignore the field/method/class, while `ACC_SYNTHETIC`+`ACC_BRIDGE` seem to mean we can't, and I don't see why. Maybe my understanding of `ACC_SYNTHETIC` semantic is wrong (?).